scripts/extensions/sams/src/components/assets/uploadAssetModal.tsx

```diff
@@ -128,7 +128,11 @@ export class UploadAssetModalComponent extends React.Component<IUploadAssetModal
 
         for (field in asset) {
             if (['_id', 'length'].includes(field) === false) {
-                data.append(field, asset[field] as string);
+                if (typeof asset[field] === 'object') {
+                    data.append(field, JSON.stringify([asset[field]]));
```

Comment:
Why is the value being supplied as an array? i.e.
```typescript
JSON.stringify([asset[field]])
```
vs
```typescript
JSON.stringify(asset[field])
```

The value being sent to the server should be the same value as stored in the local Asset, in a jsonified string

Reply:
This code is very specific to `tags`. If the value is an `object`, `JSON.stringify` should suffice for all data types. `tags` should always be an array, so it should work here without encasing in `[` and `]`